libvast/src/system/node.cpp

```diff
@@ -391,6 +391,14 @@ caf::behavior node(node_actor* self, std::string name, path dir,
   self->set_down_handler([=](const down_msg& msg) {
     VAST_DEBUG(self, "got DOWN from", msg.source);
     auto component = caf::actor_cast<caf::actor>(msg.source);
+    auto type = self->state.registry.find_type_for(component);
+    VAST_ASSERT(type != nullptr); // We monitor all components in the registry.
```

Comment:
How do you feel about *"The node monitors all components in the registry"*? I'm trying to make it clear who is the monitoring entity, because the component registry is just a container.